lobbyboy/utils.py

```diff
@@ -273,3 +274,12 @@ def try_load_key_from_file(
     if not (pri_key and pub_key) and raise_error:
         raise FileNotFoundError(f"ssh key pair not found: {private_key_file}/{public_key_file}")
     return pri_key, pub_key
+
+
+def get_cls(cls_path: str = ""):
```

Comment:
```suggestion
def import_class(cls_path: str = ""):
```

Reply:
Looks like this is reverted?

Reply:
seems so, let it be, i'll commit it later.